Add more detail about SBD_DELAY_START

tahliar opened the pull request:
### PR creator: Description

Added some more detail about this parameter, plus a link to the TID.


### PR creator: Are there any relevant issues/feature requests?

* bsc#1210435
* jsc#DOCTEAM-970


### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP5
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
- SLE-HA 12
  - [x] 12 SP5
  - [ ] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

tahliar commented:
Rendered text:

![Screenshot from 2023-09-14 16-00-39](https://github.com/SUSE/doc-sleha/assets/3069029/76b317d2-dbd0-42c2-9eba-7b56b28e1e6f)

gao-yan commented on xml/ha_storage_protection.xml:
```diff
@@ -685,7 +685,17 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
+      If you enable <parameter>SBD_DELAY_START</parameter>, you must also check the SBD service file
+      to ensure that the value of <literal>TimeoutSec</literal> is greater than the value of
+      <parameter>SBD_DELAY_START</parameter>. For more information, see
+      <link xlink:href="https://www.suse.com/support/kb/doc/?id=000019356"/>.
```
BTW, actually SBD_DELAY_START can be enabled/configured for the similar purpose in diskless SBD as well:

https://github.com/SUSE/doc-sleha/blob/main/xml/ha_storage_protection.xml#L932-L955

In that case, a setting SBD_DELAY_START=yes will take a delay value `SBD_WATCHDOG_TIMEOUT *2` rather than `msgwait`. Alternatively it can also be specified with an explicit time value.

tahliar replied:
Thanks Yan, I've now added this!

gao-yan reviewed (commented):
Thanks for taking care, @tahliar !

tahliar commented on xml/ha_storage_protection.xml:
```diff
@@ -685,7 +685,17 @@ SBD_WATCHDOG_TIMEOUT=5</screen>
+    </para>
+    <para>
+      If you enable <parameter>SBD_DELAY_START</parameter>, you must also check the SBD service file
+      to ensure that the value of <literal>TimeoutSec</literal> is greater than the value of
```
Thanks for this link, it's got more detail than whatever doc I looked at!

tahliar commented:
I've added this info to the diskless SBD procedure as well. It now looks like this: 

![Screenshot from 2023-09-15 15-17-15](https://github.com/SUSE/doc-sleha/assets/3069029/aed1d5a6-1e87-476b-8fb0-a55c0234af93)

I also noticed that step 1 in procedure 13.4 looked weird, as it said that SBD_DEVICE wasn't needed, but then the very next step said to edit SBD_DEVICE. I suspect it was somehow copied from the diskless SBD procedure. I've now fixed it so it makes sense: 

![Screenshot from 2023-09-15 15-16-51](https://github.com/SUSE/doc-sleha/assets/3069029/b2b4cf8e-7d07-4ba6-a864-b456e4ad9af7)

gao-yan reviewed (approved):
Looks good. Thanks, @tahliar !
